src/RichText.vue

```diff
@@ -99,7 +105,7 @@ export default {
 			// just return plain string nodes as text
 			if (!matches || list[index] === list[index - 1]) {
 				const text = entry.replace(/^\{[a-z\-_0-9]+\}/i, '')
-				return prepareTextNode({ h, context }, text)
+				return prepareTextNode({ h, context }, text, context.props.linkComponent)
```

Comment:
context is already passed to prepareTextNode, so I would rather add the prop access inside there when calling parseUrl

Reply:
Right, didn't notice that